src/assets/metadata/aws.json

```diff
@@ -1,296 +1,3731 @@
+              "displayName": "No device",
+              "name": "no_device",
+              "type": "Boolean",
+              "url": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/launch_configuration.html#no_device"
+            }
+          ],
+          "description": "Additional EBS block devices to attach to the instance.",
+          "displayName": "EBS block device",
+          "name": "ebs_block_device",
+          "type": "Object",
+          "url": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/launch_configuration.html#ebs_block_device"
+        },
+        {
+          "attributes": [
+            {
+              "description": "The name of the block device to mount on the instance.",
+              "displayName": "Device name",
+              "name": "device_name",
+              "required": "true",
+              "type": "String",
+              "url": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/launch_configuration.html#ephemeral_block_device"
+            },
+            {
+              "description": "Whether the device in the block device mapping of the AMI is suppressed.",
+              "displayName": "No device",
+              "name": "no_device",
+              "type": "Boolean",
+              "url": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/launch_configuration.html#ephemeral_block_device"
+            },
+            {
+              "description": "The Instance Store Device Name.",
+              "displayName": "Virtual name",
+              "name": "virtual_name",
+              "type": "String",
+              "url": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/launch_configuration.html#virtual_name"
+            }
+          ],
+          "description": "Customize Ephemeral (also known as \"Instance Store\") volumes on the instance.",
+          "displayName": "Ephemeral block device",
+          "name": "ephemeral_block_device",
+          "type": "Object",
+          "url": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/launch_configuration.html#ephemeral_block_device"
+        },
+        {
+          "attributes": [
+            {
+              "description": "The state of the metadata service: enabled, disabled.",
+              "displayName": "HTTP endpoint",
+              "name": "http_endpoint",
+              "rules": {
+                "values": [
+                  "enabled",
+                  "disabled"
+                ]
+              },
+              "type": "String",
+              "url": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/launch_configuration.html#http_endpoint"
+            },
+            {
+              "description": "If session tokens are required: optional, required.",
```

Comment:
not clear, specify what is required and/or optional ?

Reply:
done